include/deal.II/matrix_free/tensor_product_kernels.h

```diff
@@ -3482,10 +3482,10 @@ namespace internal
         constexpr unsigned int         array_size = length > 0 ? length : 1;
         std::array<Number, array_size> shape_values_x;
         std::array<Number, array_size> shape_derivs_x;
-        for (unsigned int i = 0; i < array_size; ++i)
+        for (unsigned int j = 0; j < array_size; ++j)
           {
-            shape_values_x[i] = shapes[i][0][0];
-            shape_derivs_x[i] = shapes[i][1][0];
+            shape_values_x[j] = shapes[j][0][0];
+            shape_derivs_x[j] = shapes[j][1][0];
```

Comment:
@kronbichler @bergbauer We needed to rename `i` to some other name, since the compiler was warning that there are two `i` shadowing each other.

Reply:
Here's the error message: https://github.com/dealii/dealii/actions/runs/4884140277/jobs/8716458324#step:10:2319

Reply:
Odd that it only complains about this now but yes this should be a different variable.